Microsoft.Toolkit.Services/PlatformSpecific/NETFramework/NetFrameworkStorageManager.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.IsolatedStorage;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Services.Core;
+
+namespace Microsoft.Toolkit.Services.PlatformSpecific.NetFramework
+{
+    internal class NetFrameworkStorageManager : IStorageManager
+    {
+        private const string FileName = "twitter_service_data.txt";
```

Comment:
this is not specific to twitter, is it? What happens when multiple services use this?

Reply:
This file stores any service credentials as long as the service keys are different. However, the IStorageManager is an internal class. So, in this file, we only store the keys used for the .NET Framework implementation